assets/i18n/vi.json

```diff
@@ -142,6 +142,7 @@
   "I18N_LIBRARY_LOADING": "Đang chạy",
   "I18N_LIBRARY_MAIN_HEADER": "Hãy tưởng tượng những gì bạn có thể học hôm nay....",
   "I18N_LIBRARY_N/A": "Không có",
+  "I18N_LIBRARY_NO_EXPLORATIONS": "Rất tiếc, không có cuộc thám hiểm để hiển thị.",
```

Comment:
Hi @MinhHuong, is this correct? (It's meant to be a translation of "Oops, there are no explorations to display.").